great_expectations/dataset/sqlalchemy_dataset.py

```diff
@@ -571,6 +572,10 @@ def __init__(
             self.dialect = import_library_module(
                 module_name="pyathena.sqlalchemy_athena"
             )
+        elif self.engine.dialect.name.lower() == "bigquery":
+            self.dialect = import_library_module(
+                module_name="pybigquery.sqlalchemy_bigquery"
+            )
```

Comment:
More of a style comment but would it be possible to clean up this `if/elif/else` branch?
```suggestion
        dialect_name: str = self.engine.dialect.name.lower()
        if dialect_name == "something":
            ...
        elif dialect_name == "bigquery":
            ...
        else:
            ...
```
Very minor but a nice boost to readability (plus we don't need to recreate a `str` each time!)